README.md

```diff
@@ -30,7 +30,9 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
 
 2. Fork the project, using the top right corner button, that states `fork`
 
-3. In the `data.json` file, add your own object to the **end of the file**. Your entry should be similar to this example:
+3. Create a file named `your-username.json` in the directory [`profiles`](/profiles) with the following content. 
```

Comment:
```suggestion
3. Create a file named `your-username.json` in the directory [`profiles`](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) with the following content. 
```

Reply:
Why change it to HTTPS? we can use a repo-relative link.

Reply:
I am not sure but it redirects to the URL mentioned anyways 🤷

Reply:
Ya Indeed, but repo-relative is much better and concise 
+ Also meet the Markdown standard.

Reply:
also, this won't work on other sites except github. Not sure how this is a markdown standard though.

Reply:
I will not fall into explaining the whole stranded thing, because it's the whole another topic for open discussion.

The only thing I will say is if GitHub has special support for this and all big the organization use this standard, so it makes much more sense to use this :).

Reply:
I am not sure, but a URL will be better IMHO 🤔

Reply:
relative one is better 💯